### DISCUSSION TITLE
function st_asgeojson(extensions.geometry) does not exist

### DISCUSSION THREAD

**Discussion by coloz** [supabase/supabase#14307] [Questions]
I have already enabled the PostGIS extension, and when calling RPC function, it prompts: function st_asgeojson(extensions.geometry) does not exist. However, when I run it directly in the SQL editor, the ST_AsGeoJSON function is available.

why?

**Comment by GaryAustin1** [comment]
Not sure.  I copied this:
```
SELECT ST_AsGeoJSON(t.*)
FROM (VALUES (1, 'one', 'POINT(1 1)'::geometry)) AS t(id, name, geom);
```
Into an SQL function and it runs from rpc.   Extensions is in my API search path.

More info likely needed.

**Comment by coloz** [comment]
The problem has been resolved and requires the use of `extensions.ST_ AsGeoJSON` is required to call this function
